source/rust_verify/src/verifier.rs

```diff
@@ -913,14 +930,21 @@ impl Verifier {
             funs.insert(function.x.name.clone(), (function.clone(), vis_abs));
         }
 
+        let mut exact_match = false;
```

Comment:
Might be good to call this `verify_function_exact_match` or similar?

Reply:
Both addressed in comments below!